src/proto/graplinc/grapl/api/graph/v1beta1/types.proto

```diff
@@ -200,19 +200,3 @@ message Lens {
   // score (absent prior to engagement-creator)
   optional uint64 score = 4;
 }
-
-// Result computed by an analyzer when it makes a detection.
-message ExecutionHit {
```

Comment:
This is replaced by the similar ExecutionHit in analyzer_sdk. This was ported over before we did GraphQuery. The structure is largely the same.